bazel.sh

```diff
@@ -46,38 +47,72 @@ version_as_number() {
   echo "${padded_version//.}"
 }
 
-action="$1"
-target="$2"
-min_xcode_version="$(version_as_number $3)"
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+  key="$1"
 
-# Dependencies
+  case $key in
+  -m|--min-xcode-version)
+    MIN_XCODE_VERSION="$(version_as_number $2)"
+    shift
+    shift
+    ;;
+  -v|--verbose)
+    VERBOSE_OUTPUT="1"
+    shift
+    ;;
+  *)
+    POSITIONAL+=("$1")
+    shift
+    ;;
+  esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
 
 if [ -n "$KOKORO_BUILD_NUMBER" ]; then
   # Move into our cloned repo
   cd github/repo
+
+  # Always enable verbose output on kokoro runs.
+  VERBOSE_OUTPUT=1
+fi
+
+if [ -n "$VERBOSE_OUTPUT" ]; then
+  verbosity_flags="-s"
+
+  # Display commands to stderr.
+  set -x
 fi
 
+ACTION="$1"
+TARGET="$2"
+
```

Comment:
To maintain backward compatibility: If `min-xcode-version` unset, set it as `"$(version_as_number $3)"` ?  However, I'm OK with a breaking change if we bump the major version number of the script.

Reply:
Will be a breaking change.